launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -186,7 +186,11 @@ QString MinecraftInstance::binRoot() const
 
 QString MinecraftInstance::getNativePath() const
 {
+#if defined(Q_OS_FREEBSD)
+    QDir natives_dir("/usr/local/lib/lwjgl/");
+#else
```

Comment:
That's not how to do it. Minecraft needs other native libraries than LWJGL, needs multiple versions of LWJGL based on game version, and natives need to be packaged properly.

If someone has a fork of MultiMC that does this, it is broken and should never be used as 'MultiMC'.

Reply:
Thank you for your feedback. I've tested my build extensively with MC Alpha versions, all the way up to 1.18.1 - but I'll have a look to see if it can be improved. 

It's not straight forward though, as FreeBSD is *not* a supported platform for LWJGL 3.x. So for MC 1.13+ to run, you need to [build your own LWJGL](https://github.com/johalun/lwjgl3-port), and use a start-up script to inject the correct libraries (instead of just specifying the java binary) in MultiMC. Later [forks](https://github.com/patrickSpaceSurfer/lwjgl3-port) added support for MC 1.14+as the LWJGL version changed again.

Reply:
I double checked the [official FreeBSD Ports repo](https://github.com/freebsd/freebsd-ports/tree/main/games/multimc) where their was support for MultiMC 0.6.11. The ports `Makefile` defined a `LWJGL_DIR` which when fully resolved, points to `/usr/local/lib/lwjgl` - which is what I used. As LWJGL 2.9.3 and earlier is the only officially supported versions for FreeBSD, the path is working fine for all MC versions up to MC 1.12.2. As I mentioned, for MC 1.13+, FreeBSD has no supported LWJGL to install, as you need to patch and compile a version yourself and use a start-up script.

Not ideal - I know - but what we have to live with for using FreeBSD. ;-) If you have any other suggestions, it would be greatly appreciated.

Reply:
This is not going to be accepted here. At non point is MultiMC anything other than a completely standalone application.
The only interface towards the system is Qt.

Depending on system LWJGL is out of the question in MultiMC - but if you want to maintain that in a disctinc fork, that's fine.